rasa_core/training/interactive.py

```diff
@@ -176,14 +175,19 @@ def send_action(endpoint,  # type: EndpointConfig
         return _response_as_json(r)
     except requests.exceptions.HTTPError:
         if is_new_action:
-            logger.warning("You have created a new action: {} "
-                           "which was not successfully executed. \n"
+            warning_questions = [{
+                "name": "warning",
+                "type": "input",
```

Comment:
well, what is this supposed to do? if you want the user to actively confirm this, then this should not be an `input` but rather 
```suggestion
                "type": "confirm",
```

Reply:
here is the problem 😅it is `input`, because `confirm` does not break long lines

Reply:
frankly users can input anything they want there

Reply:
oh god - this freaking bug...

Reply:
yep, so ready to merge? But to be honest for this type of warning, I think it is better

Reply:
the only reason we do it as question, is because otherwise logs get scrolled and it is easy to miss this warning, which is quite important